src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -96,6 +96,9 @@
   private int commandId;
   private final AtomicBoolean segmentFailed;
   private final UUID leaderElectionId;
+  private final AtomicBoolean failedNotified = new AtomicBoolean(false);
+  private final AtomicBoolean successNotified = new AtomicBoolean(false);
```

Comment:
thoughts about replacing these two above with just `failedOrSuccessNotified` ?

Reply:
I made this change.